app/models/classes/format_url.rb

```diff
@@ -0,0 +1,18 @@
+class Classes::FormatUrl
```

Comment:
Yeah we were really confused as to where it should go but I think PostHelpers sounds like a good place as it works directly on the post in the html.erb file. We'll move it now. Thanks